classification/feature_generation/file_iterator_test.py

```diff
@@ -64,11 +46,13 @@ def log_mem():
 
 
 def test_read_files_from_gcs():
-    path = "gs://world-fishing-827/data-production/classification/release-0.1.2/pipeline/output/features/mmsis/part-00000-of-00001.txt"
+    path = "gs://world-fishing-827/data-production/classification/release-0.1.2/pipeline/output/ids/part-00000-of-00001.txt"
```

Comment:
@bitsofbits can we move `gs://world-fishing-827/data-production/classification/release-0.1.2/` to somewhere else? something like `gfw-repositories-assets-vessel-classification/classification/release-3.0.0/`?

Reply:
@enriquetuya : This is very old, so it's really only kept around to have some consistent data for testing.  Unsharded features live `pipe-production-X` and sharded features are ephemeral. Given that we can copy just a subset of these somewhere to use for testing. Do you have a preference for what it's called?